README.md

```diff
@@ -15,17 +15,19 @@ Add real secrets to `.env`
 Source the secrets on your local machine to set the env vars:
 `source .env`
 
-1. Start with the script called `step_1.py` in `scripts/python`. <br />
-This script should be run LOCALLY. It's purpose is to create the required project level environment variables needed for the migration in the given Circle-CI On-Prem project. The environment variables set contain AWS keys and CircleCI API keys, among other things. These secrets will not be moved to SaaS.
+Install the dependencies needed to run the scripts locally:
+`pip install -r requirements.txt`
 
-**NOTE** If environment variables with the same are already set for the project, this script will OVERWRITE them. (Each environment variable in the `.env` file is prefixed with `MIGRATION_` to help with this, but can be randomized further if needed.)
+**NOTE** Anytime you want to migrate a new project, remember to change the value of `CIRCLE_PROJECT_REPONAME` in the `.env` and resource. All scripts below should be run from the root of the project. 
```

Comment:
Might also want to say "source the `.env` file"

Reply:
I think we should move the instructions to "update `CIRCLE_PROJECT_REPONAME`" to it's own step, since that is something you do for each repo. Like, the initial stuff "clone the repo; rename .env-test; add real secrets to .env" -- those are the base setup and done once for the duration of n projects you are migrating.
But then for each project you migrate, you 1. update the reponame var & source the .env file, 2. run step_1.py, etc